vignettes/customise_query.Rmd.src

```diff
@@ -0,0 +1,261 @@
+For example OSM uses [`zh_pinyin`](https://wiki.openstreetmap.org/w/index.php?title=Multilingual_names#China) instead of `zh-Latn-pinyin` for [Hanyu Pinyin](https://en.wikipedia.org/wiki/Pinyin).
+It might therefore be helpful to consult the details page of the target country on openstreetmap.org in order to see which language tags are actually used.
+In any case, neither the OpenCage API nor the functions in this package will validate the language tags you provide.
+
+For further details see [OpenCage's API documentation](https://opencagedata.com/api#language) on that subject.
+
+### Annotations
+
+OpenCage supplies additional information about the result location in the so-called [annotations](https://opencagedata.com/api#annotations).
+They include, among others, country information, time of sunset and sunrise, [UN M49](https://en.wikipedia.org/wiki/UN_M49) codes or the location in different geocoding formats, like [Maidenhead](https://en.wikipedia.org/wiki/Maidenhead_Locator_System), [Mercator projection](https://en.wikipedia.org/wiki/Mercator_projection) ([EPSG:3857](https://epsg.io/3857)), [geohash](https://en.wikipedia.org/wiki/Geohash) or [what3words](https://en.wikipedia.org/wiki/What3words).
+Some annotations, like the [Irish Transverse Mercator](https://en.wikipedia.org/wiki/Irish_Transverse_Mercator) (ITM, [EPSG:2157](https://epsg.io/2157)) or the [Federal Information Processing Standards](https://en.wikipedia.org/wiki/Federal_Information_Processing_Standards) (FIPS) code will only be shown when appropriate.
+
+Whether the annoations are shown, is controlled by the `no_annotations` argument.
+It is `TRUE` by default, which means that the output will _not_ contain annotations.
+(Yes, inverted argument names are confusing, but we just follow OpenCage's lead here.)
+When you set `no_annotations` to `FALSE`, all columns are returned (i.e. `output` is implicitly set to `"all"`).
+
+```{r annotations}
+oc_forward_df("Dublin", no_annotations = FALSE)
+```
+
+### `roadinfo`
+
+`roadinfo` indicates whether the geocoder should attempt to match the nearest road (rather than an address) and provide additional road and driving information.
+It is `FALSE` by default, which means OpenCage will not attempt to match the nearest road. Some road and driving information is nevertheless provided as part of the annotations (see above), if `roadinfo` is set to `FALSE`.
+
+```{r roadinfo}
+oc_forward_df(placename = c("Europa Advance Rd", "Bovoni Rd"), roadinfo = TRUE)
+```
+
+A [blog post](https://blog.opencagedata.com/post/new-optional-parameter-roadinfo) provides more details.
+
+### Abbreviated addresses
+
+The geocoding functions also have an `abbr` parameter, `FALSE` by default.
+When it is `TRUE` the addresses in the `formatted` field of the results are abbreviated (e.g. "Main St." instead of "Main Street").
```

Comment:
```suggestion
When it is `TRUE`, the addresses in the `formatted` field of the results are abbreviated (e.g. "Main St." instead of "Main Street").
```